### ISSUE TITLE
Overlay updating bug

### ISSUE THREAD

**Issue by theckhd** [Judgy53/OverlayPlugin#1]
Recently, the overlay started exhibiting a weird updating bug. During combat, the "Time" indicator keeps incrementing properly, but everything else (DPS, as well as the list of actors) refuses to update until combat is over. Once combat ends, the entire parse shows up properly in the overlay. 

This bug occurred on my laptop some months ago, but I never had time to troubleshoot it. Then yesterday, it started happening on my desktop. I'm not aware of anything I did to change it, nor have I been messing with the settings recently. 

The only customization I've made was editing the miniparse.html files in \resources\ to customize the columns that are displayed, but that was many months ago.

Any ideas?

**Comment by Judgy53** [comment]
Does the main ACT window also have this issue ? If not, I'm not sure why it would delay like that. If the "Time" increments fine, then the plugin should have grabbed the latest data, because everything is done at the same time. Might be ACT itself sending the wrong data somehow, or the history DB being so large it fails to detect which encounter to get the data from.

I've also noticed the original plugin have received a good bunch of updates recently, wouldn't hurt to includes them in. Unfortunately, I haven't been very motivated with SWL recently, so I can't say this'll get done soon.

In the mean time, there's not many things you can do, except maybe a full reinstall of ACT (including the config stuff in `%APPDATA%\Advanced Combat Tracker`.

**Comment by theckhd** [comment]
Nope, the main ACT window works fine. It just seems to be the communication with the plugin that's borked. The "time" field in the header updates, bu tthe "DPS" field is stuck at 0 and the data rows won't show up. After combat ends they appear, sometimes with a small delay.

It happens even on the first encounter after running ACT. If I get some time I'll try a full reinstall and see if that fixes it.

**Comment by theckhd** [comment]
A full uninstall/reinstall (including %APPDATA% content) fixed it on my laptop. I'll just have to do the same on my desktop.

**Comment by theckhd** [comment]
Just checked on desktop: simply deleting the %appdata% content was enough. No uninstall needed.
